static/app/views/insights/cache/views/cacheLandingPage.tsx

```diff
@@ -134,7 +134,7 @@ export function CacheLandingPage() {
     isFetching: isTransactionDurationFetching,
   } = useMetrics(
     {
-      search: `transaction:[${transactionsList.map(({transaction}) => `"${transaction}"`).join(',')}]`,
+      search: `transaction:[${transactionsList.map(({transaction}) => `"${transaction.replaceAll('"', '\\"')}"`).join(',')}]`,
```

Comment:
🤔 `useMetrics` uses `MutableQuery` under the hood, which should escape double quotes. Is that not working?

Reply:
We ended up having to bypass the whole `MutableQuery` thing, because it didn't handle having spaces in the transaction name correctly.
see https://github.com/getsentry/sentry/pull/70363#discussion_r1592903273

Reply:
`useMetrics` `search` is of type `search?: MutableSearch | string`